### PR TITLE
Fixing gettext bug

### PR DESCRIPTION
**Pull request by matthewdylan**
updates the minified bundle file to directly reference gettext (instead of calling it with django.gettext) for django 3.2 upgrade

SMELL ALERT: Because of some outdated packages, I wasn't able to build the bundle without more work put into getting django-jet updated (this includes moving to completely new packages, in some instances, since some are no longer accessible (see: [perfect-scrollbar](https://github.com/noraesae/perfect-scrollbar-bower)). Luckily this was such a minor change, that I was able to make an update to the bundled file itself (not optimal, of course). Once we have a more solid idea of which frontend framework to use actively on the admin side of the site (react vs. vue), we'd do well to make our own tools with that framework to replace the tools we're using from django-jet.

### CODE REVIEW

File: jet/dashboard/templates/jet.dashboard/modules/recent_actions.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
                     {% endif %}
 
                     {% if entry.content_type %}
-                        <span>{% filter capfirst %}{{ entry.content_type }}{% endfilter %}</span>
+                        <span>{% filter capfirst %}{{ entry.content_type.name }}{% endfilter %}</span>
```
Comment (ashley-hebler) [inline jet/dashboard/templates/jet.dashboard/modules/recent_actions.html:28]: So that's how you fixed that recent actions labels! Nice find

Comment (matthewdylan) [reply jet/dashboard/templates/jet.dashboard/modules/recent_actions.html:28]: yeah the default str returned for content_type changed, so switched to .name to get the same result

File: jet/static/jet/js/build/bundle.min.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
...turn!1;e(this).addClass("ui-state-active"),n.buttonElement.attr("aria-pressed","true");var t=n.element[0];s(t).not(t).map(function(){return e(this).button("widget")[0]}).removeClass("ui-state-active").attr("aria-pressed","false")}):(this.buttonElement.bind("mousedown"+this.eventNamespace,function(){return!r.disabled&&(e(this).addClass("ui-state-active"),t=this,void n.document.one("mouseup",function(){t=null}))}).bind("mouseup"+this.eventNamespace,function(){return!r.disabled&&void e(this).removeClass("ui-state-active")}).bind("keydown"+this.eventNamespace,function(t){return!r.disabled&&void(t.keyCode!==e.ui.keyCode.SPACE&&t.keyCode!==e.ui.keyCode.ENTER||e(this).addClass("ui-state-active"))}).bind("keyup"+this.eventNamespace+" blur"+this.eventNamespace,function(){e(this).removeClass("ui-state-active")}),this.buttonElement.is("a")&&this.buttonElement.keyup(function(t){t.keyCode===e.ui.keyCode.SPACE&&e(this).click()})),this._setOption("disabled",r.disabled),this._resetButton()},_determineButtonType:function(){var e,t,i;this.element.is("[type=checkbox]")?this.type="checkbox":this.element.is("[type=radio]")?this.type="radio":this.element.is("input")?this.type="input":this.type="button",
```
Comment (ashley-hebler) [inline jet/static/jet/js/build/bundle.min.js:2]: ok yeah I see what you mean with the misleading diff. I checked this in a different diff tool and this makes sense based on our discussion
Screen Shot 2022-04-05 at 1 25 06 PM

**Review by ashley-hebler — APPROVED** [review]
LGTM! Agreed that it's a necessary workaround for now and indeed exposes another good reason for us to rethink this admin dependency

File: jet/static/jet/js/build/bundle.min.js (hunk 2 above)
Comment (matthewdylan) [reply jet/static/jet/js/build/bundle.min.js:2]: oh, sweet... I'll make note of that differ... very helpful

**Pull request merged**